app/services/blog_post_categorizer.rb

```diff
@@ -0,0 +1,17 @@
+class BlogPostCategorizer
+  def technology_for(blog_post_payload)
+    Technology.all.detect(default_technology) do |technology|
```

Comment:
why not something like:
```suggestion
    Technology.other.map do |technology|
```

looks simpler to me

Reply:
But is that actually equivalent? Doesn't `Technology.other` return a Technology object?
The code I wrote there means to look for the technology that matches any of the tags or categories. If none is found, then the default technology will be returned.

Reply:
I thought `other`, was a scope 😂 sorry

Reply:
Haha no worries! As much as I love the `detect` method, its expressiveness is far from being good 👎